src/hotspot/share/gc/shenandoah/shenandoahControlThread.hpp

```diff
@@ -90,7 +90,7 @@ class ShenandoahControlThread: public ConcurrentGCThread {
   ShenandoahSharedFlag _do_counters_update;
   ShenandoahSharedFlag _force_counters_update;
   GCCause::Cause       _requested_gc_cause;
-  GenerationMode       _requested_generation;
+  ShenandoahGenerationType       _requested_generation;
```

Comment:
White space got a little weird here.

Reply:
Good spot, fixed. (In not yet published commit.)